static/css/footer.less

```diff
@@ -6,6 +6,29 @@
     margin-bottom: 1em;
 }
 
+.futuremakers .cta button {
```

Comment:
This doesn't belong in footer.less. Since it's specific to future makers, I'd be inclined to just create a separate less file for whatever future makers magic you need to do.